bridge_adaptivity/module/tasks.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import, unicode_literals
+
+from celery.task import task
+
+
+@task()
+def sync_collection_engines(collection_id=None, created_at=None):
+    from module.models import Collection
```

Comment:
why don't you use import on module level?

Reply:
Because of cycling import, unfortunately, we use this task in the module.models file.